<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>741</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-bottom-right-radius:50px;
border-top-left-radius:50px;
background-color: rgb(27, 19, 57);

</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>231</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-bottom-right-radius:50px;
border-top-left-radius:50px;
background-color: rgb(27, 19, 57);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Images/GestionStock.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>521</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Snap ITC</family>
      <pointsize>28</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(181, 120, 255);
font: 28pt &quot;Snap ITC&quot;;
</string>
    </property>
    <property name="text">
     <string>Sign UP 
 Fill in the form</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernamefield">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 19, 57);
color: rgb(255, 230, 250);


border:none;
border-bottom :2px solid ;
border-bottom-color: rgb(255, 170, 255);
padding-bottom:7px;


</string>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordfield">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 19, 57);
color: rgb(255, 230, 250);



border:none;
border-bottom :2px solid ;
border-bottom-color: rgb(255, 170, 255);
padding-bottom:7px;


</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Password </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_signup">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>380</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_signup{

	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(181, 120, 255), stop:1 rgb(255, 170, 255));

	color:rgba(255, 255, 255, 210);

	border-radius:5px;

}




QPushButton#btn_signup:hover{

	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(181, 120, 255), stop:1 rgb(255, 170, 255));

}



QPushButton#lbtn_signup:pressed{

	padding-left:5px;

	padding-top:5px;

	background-color:rgb(251, 130, 255);

}



QPushButton#btn_signup_2, #btn_signup_3, #btn_signup_4, #btn_signup_5{

	background-color: rgba(0, 0, 0, 0);

	color:rgb(251, 130, 255);

}



QPushButton#login_btn_2:hover, #login_btn_3:hover, #login_btn_4:hover, #login_btn_5:hover{

	color: rgb(251, 130, 255);

}



QPushButton#login_btn_2:pressed, #login_btn_3:pressed, #login_btn_4:pressed, #login_btn_5:pressed{

	padding-left:5px;

	padding-top:5px;

	color:rgb(251, 130, 255);;

}
</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ic/icons8-ajouter-32.png</normaloff>ic/icons8-ajouter-32.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailfield">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 19, 57);
color: rgb(255, 230, 250);



border:none;
border-bottom :2px solid ;
border-bottom-color: rgb(255, 170, 255);
padding-bottom:7px;


</string>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="C_passwordfield">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 19, 57);
color: rgb(255, 230, 250);



border:none;
border-bottom :2px solid ;
border-bottom-color: rgb(255, 170, 255);
padding-bottom:7px;


</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="telephonefield">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 19, 57);
color: rgb(255, 230, 250);



border:none;
border-bottom :2px solid ;
border-bottom-color: rgb(255, 170, 255);
padding-bottom:7px;


</string>
    </property>
    <property name="placeholderText">
     <string>Téléphone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="first_namefield">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 19, 57);
color: rgb(255, 230, 250);



border:none;
border-bottom :2px solid ;
border-bottom-color: rgb(255, 170, 255);
padding-bottom:7px;



</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>First name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="last_namefield">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 19, 57);
color: rgb(255, 230, 250);



border:none;
border-bottom :2px solid ;
border-bottom-color: rgb(255, 170, 255);
padding-bottom:7px;


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>Last name</string>
    </property>
   </widget>
   <widget class="QLabel" name="error5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="error4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="error3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="error2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="error1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="error7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="error9">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>460</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="error6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>500</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="error41">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="error71">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ic/icons8-nom-32.png</normaloff>ic/icons8-nom-32.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ic/icons8-utilisateur-masculin-64.png</normaloff>ic/icons8-utilisateur-masculin-64.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ic/icons8-email-64.png</normaloff>ic/icons8-email-64.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ic/icons8-mot-de-passe-100.png</normaloff>ic/icons8-mot-de-passe-100.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>443</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ic/icons8-mot-de-passe-32.png</normaloff>ic/icons8-mot-de-passe-32.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ic/icons8-téléphone-32.png</normaloff>ic/icons8-téléphone-32.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ic/icons8-nom-32.png</normaloff>ic/icons8-nom-32.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>75</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ic/icons8-sortie-64.png</normaloff>ic/icons8-sortie-64.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
